docs/RELEASE-NOTES.rst

```diff
@@ -5,6 +5,24 @@
 Release Notes for F5 Openstack Agent
 ====================================
 
+v9.9.30 (Pike, Queens)
+--------------------------------------------
+NOTE: This version of F5 Openstack Agent supports Pike and Queens Openstack releases.
+
+Added Functionality
+```````````````````
+* Add flavor support: connection limit and connection rate limit.
+* Bind logging profile to http/terminated_https vs.
+* Add checking status support and don't send the members with status of none or checking.
```

Comment:
There is a new functionality: Support cipher options 
https://github.com/F5Networks/f5-openstack-agent/pull/1707